tests/dualtor/test_t1_to_standby.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import random
+
+from scapy.all import IP, Ether
+from ptf import testutils, mask
+from tests.common.dualtor.dual_tor_mock import *
+from tests.common.dualtor.tunnel_traffic_utils import tunnel_traffic_monitor
+from tests.common.dualtor.dual_tor_utils import get_t1_ptf_ports
+from tests.common.dualtor.dual_tor_utils import rand_selected_interface
+from tests.common.dualtor.dual_tor_utils import flush_neighbor
+from tests.common.dualtor.server_traffic_utils import ServerTrafficMonitor
+
+
+pytestmark = [
+    pytest.mark.topology("t0")
+]
+
+
+@pytest.fixture(scope="function")
+def build_packet_to_server(ptfadapter, rand_selected_interface, rand_selected_dut, tunnel_traffic_monitor):
+    """Build packet destinated to server."""
+    tor = rand_selected_dut
+    _, server_details = rand_selected_interface
+    server_ipv4 = server_details["server_ipv4"].split("/")[0]
+    pkt_dscp = random.choice(range(0, 33))
+    pkt_ttl = random.choice(range(3, 65))
+    pkt = testutils.simple_ip_packet(
+        eth_dst=tor.facts["router_mac"],
+        eth_src=ptfadapter.dataplane.get_mac(0, 0),
+        ip_src="1.1.1.1",
+        ip_dst=server_ipv4,
+        ip_dscp=pkt_dscp,
+        ip_ttl=pkt_ttl
+    )
+    logging.info("the packet destinated to server %s:\n%s", server_ipv4, tunnel_traffic_monitor._dump_show_str(pkt))
+    return pkt
+
+
+def test_standby_tor_remove_neighbor_downstream_standby(
+    apply_mock_dual_tor_tables,
+    apply_mock_dual_tor_kernel_configs,
+    apply_standby_state_to_orchagent,
+    build_packet_to_server,
+    conn_graph_facts, ptfadapter,
+    rand_selected_dut, rand_selected_interface,
+    tunnel_traffic_monitor, tbinfo, vmhost
+):
+    """
+    @summary: Verify that after remove neighbor entry for a server over standby
+              Tor, the packets sent to the server will be dropped(neither passed
+              to server or redirected to the active ToR).
+    """
+
+    def build_exp_pkt_to_server(packet):
+        exp_pkt = mask.Mask(packet)
+        exp_pkt.set_do_not_care_scapy(Ether, "dst")
+        exp_pkt.set_do_not_care_scapy(Ether, "src")
+        exp_pkt.set_do_not_care_scapy(IP, "tos")
+        exp_pkt.set_do_not_care_scapy(IP, "ttl")
+        exp_pkt.set_do_not_care_scapy(IP, "chksum")
+        return exp_pkt
+
+    tor = rand_selected_dut
+    pkt = build_packet_to_server
+    iface, server_details = rand_selected_interface
+    server_ipv4 = server_details["server_ipv4"].split("/")[0]
+
+    exp_pkt = build_exp_pkt_to_server(pkt)
+
+    ptf_t1_intf = random.choice(get_t1_ptf_ports(tor, tbinfo))
+    logging.info("send traffic to server %s from ptf t1 interface %s", server_ipv4, ptf_t1_intf)
+    with flush_neighbor(tor, server_ipv4):
+        with tunnel_traffic_monitor(tor, existing=False):
+            with ServerTrafficMonitor(tor, vmhost, iface, conn_graph_facts, exp_pkt, existing=False):
+                testutils.send(ptfadapter, int(ptf_t1_intf.strip("eth")), pkt, count=10)
```

Comment:
How did we decide to send 10 packets and not more? I think 10 packets are too less to verify close to real operational scenario.